.circleci/config.yml

```diff
@@ -100,7 +100,7 @@ jobs:
             npm ci
             npm test
             mkdir -p coverage-alpine
-            cp coverage/* coverage-alpine/
+            cp -r coverage/* coverage-alpine/
```

Comment:
Note that the `*` (glob) used on [line 103](https://github.com/codecov/uploader/pull/186/files/54e0992fcfc063877ae93c4e30858ef5d15cbd10..f6c38e0a43e9ea6c8224fcb2c0aa16ae651d0d3e#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R103) will expand to *only* include files that do *not* begin with a dot in Bash. In other words, if any files begin with a period, they will not be copied over to the destination.

I'm not sure how pertinent that information is but it may be relevant in the future if it's ever expected for a `.metadata` file or something to be copied.

Reply:
I don't think it matters, but there is always the possibility of us switching the coverage reporter and wondering why our CI fails :)